src/views/Project/Details/General/Dashboard/index.js

```diff
@@ -33,7 +33,12 @@ const propTypes = {
 
 const defaultProps = {
     className: '',
-    projectDashboard: {},
+    projectDashboard: {
```

Comment:
leadActivity and entriesActivity should be checked for undefined values where it is used